### PR TITLE
Fix to issue #33, 'fixed' pull request #35

### PR DESCRIPTION
**Pull request by Grimitsu**
Fixed issue #33 throwing BadQueryException. Some videos have keys xtag or axtags, which has value containing '=' on first attempt to get video. E.g. `axtags=tx=9417362,` though next attemps to get video contain no value value for this key i.e `axtags=`.

Also fixed KeyNotFoundException, two guys already tried to fix it #35 but that's not it.when video page provide no info about adaptive formats, i.e. there is no `adaptive_fmts` key instead there is a link to a MPD manifest with links to this videos and info about them.

[Change comparison](https://github.com/jamesqo/libvideo/compare/master...Grimitsu:master)

### CODE REVIEW

File: src/libvideo/Helpers/Html.cs [hunk 1]
```diff
@@ -14,5 +14,19 @@ internal static class Html
...
+            string closing = "</BaseURL";
+            int start = source.IndexOf(opening);
+            if (start != -1)
+            {
+                string temp = source.Substring(start);
+                var Uris = temp.Split(new string[] { opening }, StringSplitOptions.RemoveEmptyEntries)
+                    .Select(v => v.Substring(0, v.IndexOf(closing)));
+                return Uris;
+            }
+            throw new NotSupportedException();
```
Comment (jamesqo) [inline src/libvideo/Helpers/Html.cs:30]: Question: Have you actually tried building/testing on your machine? It appears this is missing a closing brace. (To build you can open the `libvideo.sln` solution in VS + run NuGet restore and build.)

**Comment by Grimitsu** [comment]
Please forgive me, James. I've messed things up trying to commit without unwanted code and comments and yes, i tried it, it works: libvideo.compat works, libvideo too. I'll fix my mistakes and make another commit.

**Comment by Grimitsu** [comment]
@jamesqo fixed syntax mistakes. Modified debug program to test libvideo and libvideo.compat. Removed Newtonsoft.Json and YoutubeExtractor referencies from debug program and added libvideo.compat reference.(YoutubeExtractor namespace)

File: src/libvideo.debug/libvideo.debug.csproj [hunk 2]
```diff
@@ -61,6 +53,10 @@
     <None Include="packages.config" />
   </ItemGroup>
   <ItemGroup>
+    <ProjectReference Include="..\libvideo.compat\libvideo.compat.csproj">
```
Comment (jamesqo) [inline src/libvideo.debug/libvideo.debug.csproj]: Sorry, could you please revert the changes to this file? I referenced `YoutubeExtractor` instead of the compatibility API on purpose so I could flesh out any bugs in this lib vs. YoutubeExtractor.

**Comment by Grimitsu** [comment]
@jamesqo reverted changes to csproj. Please check [13422cf](https://github.com/jamesqo/libvideo/pull/39/commits/13422cf1c434e7d35dabc65d4def01681264554b)

**Comment by jamesqo** [comment]
Merged in a new branch: https://github.com/jamesqo/libvideo/tree/merge-39

I'll publish the changes sometime tomorrow. Thanks!
